src/components/RecoveryWordsInput.css

```diff
@@ -0,0 +1,168 @@
+.recovery-words-input {
+    width: 100%;
+    max-width: 360px;
+    padding: 16px;
+    margin-top: 24px;
+    box-sizing: border-box;
+    position: relative;
+    background: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAJElEQVQYV2NkwAT/GdHE/jMwMDAiC4IFQIpggnABmCCKAEgQAABOBQUaykC8AAAAAElFTkSuQmCC');
+    background-color: rgba(0, 0, 0, 0.4);
+    border: 1px solid white;
+    border-radius: 8px;
+    color: white;
+    height: 250px;
+}
+
+.recovery-words-input .label {
+    display: block;
+    position: absolute;
+    width: 100%;
+    padding-right: 4rem;
+    box-sizing: border-box;
+    top: -1.666rem;
+    font-size: 1.5rem;
+    line-height: 3.25rem;
+    transition: opacity 300ms;
+}
+
+.recovery-words-input .label > div {
+    display: inline-block;
+    border-radius: 3rem;
+    background: url('data:image/svg+xml;base64,PHN2ZyBmaWxsPSIjZmZmIiBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZD0iTTE4IDhoLTFWNmMwLTIuNzYtMi4yNC01LTUtNVM3IDMuMjQgNyA2djJINmMtMS4xIDAtMiAuOS0yIDJ2MTBjMCAxLjEuOSAyIDIgMmgxMmMxLjEgMCAyLS45IDItMlYxMGMwLTEuMS0uOS0yLTItMnptLTYgOWMtMS4xIDAtMi0uOS0yLTJzLjktMiAyLTIgMiAuOSAyIDItLjkgMi0yIDJ6bTMuMS05SDguOVY2YzAtMS43MSAxLjM5LTMuMSAzLjEtMy4xIDEuNzEgMCAzLjEgMS4zOSAzLjEgMy4xdjJ6Ii8+PC9zdmc+');
+    background-size: 14px;
+    background-position: 9px center;
+    background-repeat: no-repeat;
+    background-color: black;
+    padding-left: 4rem;
+    padding-right: 2rem;
+}
+
+.recovery-words-input .recovery-words-input-field {
```

Comment:
Are we still using this animation?

Reply:
yep, the fields slide in "on show". I'd say it's optional